config/cloud/blogs/blog-anthos-gke-acm-bootstrap.json

```diff
@@ -2,9 +2,30 @@
   "expertise": "INTERMEDIATE",
   "version": 1,
   "source": "other",
-  "authorIds": ["paulwilljones"],
+  "authorIds": [
+    "paulwilljones"
+  ],
   "author": "Paul Jones",
   "title": "Bootstrap GKE with Anthos | Jetstack Blog",
   "link": "https://www.jetstack.io/blog/anthos-gke-acm-bootstrap/",
```

Comment:
@gkindavishal : this project has been already submitted by Paul (no advocu data on Paul's submissions)

Profile: https://devlibrary.withgoogle.com/authors/paulwilljones

Reply:
Sorry, didn't get you. This is Paul's article - no change, only formatting updated.